support/version/version.go

```diff
@@ -0,0 +1,31 @@
+package version
+
+import (
+	"fmt"
+)
+
+var (
```

Comment:
What keeps us from getting this info from go to avoid the ldflags? https://github.com/openshift/hypershift/pull/1324/files#diff-73b8ec8a05213857da6e9598e34c5d7a18a6e5349250908ae6710578236977f2

Reply:
/hold
At minimum, we should not end up with two `version` packages

Reply:
That makes sense to me. I did not realize we could already get the revision. Will update to use that.

Reply:
updated